### DISCUSSION TITLE
React sample shows compile error

### DISCUSSION THREAD

**Discussion by gangtao** [perspective-dev/perspective#2507] [Q&A]
Hi,

This is really a cool project and I am starting exploring.

Those examples seems kind of out of date. I have managed to make esbuild running by helping from @texodus , while webpack and react demo still have issues

for react demo, can run but showing following compile warnings

```

Compiled with problems:
×
WARNING in ./node_modules/@finos/perspective-viewer/dist/esm/perspective-viewer.js 2:48999-49000
Should not import the named export 'version' (imported as 'Q') from default-exporting module (only default export is available soon)
WARNING in ./node_modules/@finos/perspective-viewer/dist/esm/perspective-viewer.js 2:49112-49113
Should not import the named export 'version' (imported as 'Q') from default-exporting module (only default export is available soon)
WARNING in ./node_modules/@finos/perspective-viewer/dist/esm/perspective-viewer.js 2:49205-49206
Should not import the named export 'version' (imported as 'Q') from default-exporting module (only default export is available soon)
```

<img width="1327" alt="image" src="https://github.com/finos/perspective/assets/3509037/dc6a8547-d7b7-44a9-93ae-f36f434607af">

not sure if this is something expected.

**Comment by texodus** [comment]
@gangtao Please stop spamming us these reports - this is not WhatsApp.

You didn't provide even the _name_ of the example you're running so I can barely speculate at what problem you're facing, but we build [both the `react-example` and `webpack-example` in CI](https://github.com/finos/perspective/actions/runs/7562181415/job/20592096459#step:18:1019) (and I just built both locally to verify). They work just fine. We _cannot_ help you with screenshots of the errors you encounter - if you want help from us, you need to create a complete stand-alone reproduction with clear instructions on how I can see what you're seeing.

**Comment by texodus** [comment]
<img width="984" alt="Screenshot 2024-01-17 at 9 35 15 PM" src="https://github.com/finos/perspective/assets/60666/c8983c5d-8fe3-4d85-a4a6-018dc4fb2f3c">
